2025-07-07 00:01:24,021 - INFO  -AWS SQS client initialized successfully.
2025-07-07 00:01:24,021 - CRITICAL  -Failed to load or validate credentials: 'key'. Exiting.
2025-07-07 00:01:32,447 - ERROR  -Error processing webhook: cannot access local variable 'success' where it is not associated with a value
Traceback (most recent call last):
  File "E:\PM -AI-TOOL\pm-ai-tool\jira_webhook\vv.py", line 216, in handle_webhook
    if success:
       ^^^^^^^
UnboundLocalError: cannot access local variable 'success' where it is not associated with a value
2025-07-07 00:02:16,449 - ERROR  -Error processing webhook: cannot access local variable 'success' where it is not associated with a value
Traceback (most recent call last):
  File "E:\PM -AI-TOOL\pm-ai-tool\jira_webhook\vv.py", line 216, in handle_webhook
    if success:
       ^^^^^^^
UnboundLocalError: cannot access local variable 'success' where it is not associated with a value
2025-07-07 00:02:20,241 - ERROR  -Error processing webhook: cannot access local variable 'success' where it is not associated with a value
Traceback (most recent call last):
  File "E:\PM -AI-TOOL\pm-ai-tool\jira_webhook\vv.py", line 216, in handle_webhook
    if success:
       ^^^^^^^
UnboundLocalError: cannot access local variable 'success' where it is not associated with a value
2025-07-07 00:02:23,778 - ERROR  -Error processing webhook: cannot access local variable 'success' where it is not associated with a value
Traceback (most recent call last):
  File "E:\PM -AI-TOOL\pm-ai-tool\jira_webhook\vv.py", line 216, in handle_webhook
    if success:
       ^^^^^^^
UnboundLocalError: cannot access local variable 'success' where it is not associated with a value
2025-07-07 00:03:47,186 - ERROR  -Error processing webhook: cannot access local variable 'success' where it is not associated with a value
Traceback (most recent call last):
  File "E:\PM -AI-TOOL\pm-ai-tool\jira_webhook\vv.py", line 216, in handle_webhook
    if success:
       ^^^^^^^
UnboundLocalError: cannot access local variable 'success' where it is not associated with a value
2025-07-07 00:04:12,939 - ERROR  -Error processing webhook: cannot access local variable 'success' where it is not associated with a value
Traceback (most recent call last):
  File "E:\PM -AI-TOOL\pm-ai-tool\jira_webhook\vv.py", line 216, in handle_webhook
    if success:
       ^^^^^^^
UnboundLocalError: cannot access local variable 'success' where it is not associated with a value
